src/main/kotlin/org/jetbrains/intellij/utils/BundledKotlinStdlib.kt

```diff
@@ -0,0 +1,33 @@
+package org.jetbrains.intellij.utils
+
+import org.jetbrains.intellij.Version
+
+private val bundledStdlibApiMap = mapOf(
```

Comment:
we can't do that. bugfix releases could change it. also what about AS or other 3rd party IDEs?

can't we scan $IDE$/lib/kotlin-stdlib.jar?